src/app/core/constants/app-constants.ts

```diff
@@ -6,4 +6,7 @@ export const constants = {
   UPDATE_NOTE: 'todoUpdate',
   LOGIN: 'login',
   SIGNUP: 'signup',
+  GET_USERS: 'users',
```

Comment:
you could refactor these urls a little, make thme snake case, so todo-delete, todo-delete-assign-user etc maybe make them shorter :D